### PR TITLE
OSDOCS-2358: release notes for custom error code response pages

### PR DESCRIPTION
**Pull request by skrthomas**
https://issues.redhat.com/browse/OSDOCS-2358

For 4.9

Known Issue: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-known-issues

Preview: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-networking

@miheer @jechen0648 ready for QA.

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -147,6 +147,13 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-nw-customize-ingress-error-pages"]
+==== Customizing HAProxy error code response pages
+
+You can specify a custom error code response page for either 503, 404, or both error pages.
```
Comment (miheer) [inline release_notes/ocp-4-9-release-notes.adoc:153]: You can specify a custom **http** error code response page for either 503, 404, or both error pages.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -705,6 +712,16 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* Cluster administrators can specify a custom error code response page for either 503, 404, or both error pages. The router does not reload to reflect custom error code pages updates. As a workaround, rsh in the router pods and run `reload-haproxy` in all the router pods that serve the custom http error code pages:
```
Comment (miheer) [inline release_notes/ocp-4-9-release-notes.adoc:715]: Cluster administrators can specify a custom **http** error code response page for either 503, 404, or both error pages

**Review by miheer — CHANGES_REQUESTED** [review]

**Comment by jechen0648** [comment]
> https://issues.redhat.com/browse/OSDOCS-2358
> 
> For 4.9
> 
> Known Issue: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-known-issues
> 
> Preview: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-networking
> 
> @miheer @jechen0648 ready for QA.

From this preview for ocp-4-8-networking, I saw under "Customizing HAProxy error code response pages" section, there is "For more information, see Customizing HAProxy error code response pages", if I click on hyperlink for "Customizing HAProxy error code response pages", it leads me to page:  https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html#nw-customize-ingress-error-pages_configuring-ingress, and in this page, I do not see any place describe how to customize HAProxy error code response pages.      Did I look at wrong place?

**Comment by jechen0648** [comment]
> > https://issues.redhat.com/browse/OSDOCS-2358
> > For 4.9
> > Known Issue: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-known-issues
> > Preview: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-networking
> > @miheer @jechen0648 ready for QA.
> 
> From this preview for ocp-4-8-networking, I saw under "Customizing HAProxy error code response pages" section, there is "For more information, see Customizing HAProxy error code response pages", if I click on hyperlink for "Customizing HAProxy error code response pages", it leads me to page: https://deploy-preview-36498--osdocs.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html#nw-customize-ingress-error-pages_configuring-ingress, and in this page, I do not see any place describe how to customize HAProxy error code response pages. Did I look at wrong place?

Thanks for clarification from Sara, this PR looks good to me.

**Review by mikemckiernan — COMMENTED** [review]
/lgtm

**Comment by skrthomas** [comment]
#33497 (RN Tracker)

**Pull request merged**